src/main/com/microsoft/azure/documentdb/changefeedprocessor/internal/PartitionObserverManager.java

```diff
@@ -68,7 +83,7 @@ public void run() {
 						} catch (ExecutionException e) {
 							e.printStackTrace();
```

Comment:
we probably should use slf4j for logging rather than printing to std err. But I am not sure if this fits in the scope of this PR or not.

Reply:
not in the scope for this release